core/controllers/community_dashboard.py

```diff
@@ -219,3 +220,19 @@ def get(self):
                 community_rights.can_review_questions
                 if community_rights else False)
         })
+
+
+class CommunityMessageHandler(base.BaseHandler):
+    """The handler for checking if community message is enabled and returns
+    community message.
+    """
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        self.render_json({
+            'community_message_enabled': (
```

Comment:
contributor_dashboard_message, not community_message. "Contributor" and "Community" are different things...

Ditto elsewhere.